framework/include/wifi_manager/wifi_manager.h

```diff
@@ -204,12 +226,24 @@ wifi_manager_result_e wifi_manager_get_info(wifi_manager_info_s *info);
 /**
  * @brief Connect to an access point.
  * @details @b #include <wifi_manager/wifi_manager.h>
- * @param[in] ssid, passphrase, authentication type, and cryto type of the access point which the wi-fi interface connect to.
+ * @param[in] config ssid, passphrase, authentication type, and cryto type of the access point which the wi-fi interface connect to.
```

Comment:
Why did you put config?

Reply:
it's doxygen rule. The name of parameter should be written before description